hack/metadata.schema.json

```diff
@@ -122,46 +122,43 @@
           "type": "array",
           "items": {
             "type": "string"
-          },
-          "required": true
+          }
         },
         "apiGroups": {
           "type": "array",
           "items": {
             "type": "string"
-          },
-          "required": true
+          }
         },
         "resources": {
           "type": "array",
           "items": {
             "type": "string"
-          },
-          "required": true
+          }
         },
         "resourceNames": {
           "type": "array",
           "items": {
             "type": "string"
           }
         }
-      }
+      },
+      "required": [ "verbs", "apiGroups", "resources" ]
     },
     "roleRule": {
       "type": "object",
       "properties": {
         "namespace": {
-          "type": "string",
-          "required": true
+          "type": "string"
         },
         "rules": {
           "type": "array",
           "items": {
             "$ref": "#/$defs/rules"
-          },
-          "required": true
+          }
         }
-      }
+      },
+      "required": [ "namespace", "rules" ]
     }
   }
-}
+}
```

Comment:
Could you add the empty line back?

Reply:
any reason we need an extra blank line in the end?

Reply:
[that's a common across our repos](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file)

Reply:
Hi, @wanghaoran1988 .
I have amended the commit.
Could you please check?